src/diffusers/schedulers/scheduling_dpmsolver_multistep.py

```diff
@@ -267,7 +267,7 @@ def set_timesteps(self, num_inference_steps: int = None, device: Union[str, torc
             sigmas = np.flip(sigmas).copy()
             sigmas = self._convert_to_karras(in_sigmas=sigmas, num_inference_steps=num_inference_steps)
             timesteps = np.array([self._sigma_to_t(sigma, log_sigmas) for sigma in sigmas]).round()
-            sigmas = np.concatenate([sigmas, sigmas[-1:]]).astype(np.float32)
```

Comment:
@patrickvonplaten 

Was there a reason to set the sigmas this way? or is it just a mistake that we didn't catch in the PR?

By repeating the last sigma twice, we make the very last step a dummy step with zero step size

i.e. this code here is `x_t= sample` because we have `sigma_t = sigma_s0` and `h=0`
```
x_t = (
                    (sigma_t / sigma_s0) * sample
                    - (alpha_t * (torch.exp(-h) - 1.0)) * D0
                    - 0.5 * (alpha_t * (torch.exp(-h) - 1.0)) * D1
                )
```
https://github.com/huggingface/diffusers/blob/774f5c45817805546ae5eb914c175d4fe72dcfe9/src/diffusers/schedulers/scheduling_dpmsolver_multistep.py#L630

Reply:
Nice catch! Was this the culprit?

Reply:
yes this was it!